data/data.json

```diff
@@ -1722,6 +1722,27 @@
       "benefitExamples": [],
       "seeMoreSentenceEn": "",
       "seeMoreSentenceFr": ""
+    },
+    {
+      "id": "recU6FYqNjRupolvb",
```

Comment:
this looks like an empty row in the benefits table - maybe this is why the tests failed?

Reply:
yeah that's why it failed the first time